crates/aptos-crypto/src/lib.rs

```diff
@@ -24,6 +24,11 @@ mod unit_tests;
 pub use self::traits::*;
 pub use hash::HashValue;
 
+// We need to add this here if we want to use aptos-crypto-derive's CryptoHasher and BCSCryptoHasher
```

Comment:
I will fix this later.

The problem (I think) lies in https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-crypto-derive/src/lib.rs#L398 and in https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-crypto-derive/src/lib.rs#L428, which generate a `use aptos_crypto::...` line that will not work when generated inside the `aptos-crypto` crate itself (e.g., like in our Ed25519 PoK implementation in `crates/aptos-crypto/src/ed25519/ed25519_pok.rs`).